spec/sample-data/valid/control-statement-03.json

```diff
@@ -9,7 +9,7 @@
   "control_level": "unknown",
   "sample_date": "2016-01-01",
   "source_url": "http://download.companieshouse.gov.uk/en_pscdata.html",
-  "retrieved_at": "2015-08-11T23:00:08+01:00",
+  "retrieved_at": "2015-08-11 23:00:08 +0100",
```

Comment:
So this actually validates as a `"format": "date"`..? It doesn't have to be `"2015-08-11"` exactly?